Src/WitsmlExplorer.Frontend/models/fluidsReport.ts

```diff
@@ -8,4 +9,5 @@ export default interface FluidsReport extends ObjectOnWellbore {
   tvd: MeasureWithDatum;
   numReport: string;
   commonData: CommonData;
+  fluids: Fluid[];
```

Comment:
`fluids` is unused. I see that some other models have their component lists specified in their interfaces without using that. Might as well remove it I guess?